### DISCUSSION TITLE
How to get Network icons?

### DISCUSSION THREAD

**Discussion by gitaccount88** [trakt/trakt-api#359] [Q&A] [answered]
Hello,
is there any to get networks (i.e, Netflix, Disney+) icons that are showed in "All Time Stats' page in 'TV SHOW NETWORKS' bubble graph?
the api of trakt only return name of networks.

[icon example](https://trakt.tv/assets/channels/paramount-plus-5cc13315e057982e439880f7a0393fb1e9ccf10670344a3d9d24ede5da188c93.png)

Thanks.

**Comment by rectifyer** [comment, marked as answer]
We don't provide any images in the Trakt API currently. I suggest using TMDB (https://developers.themoviedb.org/3/networks/get-network-images) to get network images.
